src/member/member.service.ts

```diff
@@ -1,92 +1,91 @@
-import { ForbiddenException, Injectable } from '@nestjs/common';
-import { PrismaService } from 'src/prisma/prisma.service';
+import {
+  Injectable,
+  NotFoundException,
+  ConflictException,
+} from '@nestjs/common';
+import { MemberRepository } from './member.repository';
+import { MemberRequestDto } from './dto/member-request.dto';
+import { MemberResponseDto } from './dto/member-response.dto';
 
 @Injectable()
 export class MembersService {
-  constructor(private readonly prisma: PrismaService) {}
+  constructor(private readonly memberRepository: MemberRepository) {}
 
-  async getMemberList(groupId: number) {
-    const members = await this.prisma.userGroup.findMany({
-      where: { groupId },
-      include: {
-        user: true,
-      },
-      orderBy: {
-        createdAt: 'asc',
-      },
-    });
+  async getMemberList(groupId: number): Promise<MemberResponseDto[]> {
+    const members = await this.memberRepository.findAllMembersByGroup(groupId);
 
     return members
       .filter((member) => member.role !== 'admin')
       .map((member) => ({
         userId: member.userId,
-        name: member.user.name,
+        name: member.user?.name ?? '',
         role: member.role,
         joinedAt: member.createdAt,
       }));
   }
 
-  async getGroupMember(groupId: number, userId: number) {
-    return this.prisma.userGroup.findFirst({
-      where: {
-        groupId,
-        userId,
-      },
-      include: {
-        user: true,
-      },
-    });
+  async getGroupMember(
+    groupId: number,
+    userId: number,
+  ): Promise<MemberResponseDto | null> {
+    const member = await this.memberRepository.findGroupMember(groupId, userId);
+    if (!member) {
+      return null;
+    }
+    return {
+      userId: member.userId,
+      name: member.user?.name ?? '',
+      role: member.role,
+      joinedAt: member.createdAt,
+    };
   }
 
-  async validateRemoveMember(
+  async processRemoveMember(
```

Comment:
```suggestion
  async removeMember(
```
개인적인 생각으로는 아래 removeMember라는 메서드를 따로 둘 필요는 없어보여요
재사용되는 코드도 아니라고 생각되고 단순히 repository 구현을 래핑한 정도로 보여서요